introduction/java_datatypes/Solution.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+
+class Solution {
+ 
+  private static void dataType(long valueToCheckType) {
+    System.out.println(valueToCheckType + " can be fitted in:");
+        if ((valueToCheckType >= -128) && (valueToCheckType <= 127)) System.out.println("* byte");
+        if ((valueToCheckType >= -32768) && (valueToCheckType <= 32767)) System.out.println("* short");
+        if ((valueToCheckType >= -Math.pow(2, 31)) && (valueToCheckType <= Math.pow(2, 31) - 1)) System.out.println("* int");
```

Comment:
This line is to long. The limit is 100 characters per line. I don't actually know how to efficiently check the line size in online editor, but this and the following lines are definitely longer :)
Please break them.